JPF/utils.py

```diff
@@ -239,7 +242,7 @@ def send_df_to_sql(df, log, description, table_name, db_action, sql_chunk_size,
     df.to_sql(name=table_name, con=conn, if_exists=action,\
               index=False, chunksize=sql_chunk_size, dtype=outputdict)
 
-    if db_action == 'replace':
+    if db_action == 'replace' and set_default_primary_key:
         conn.execute("ALTER TABLE %s ADD id INT PRIMARY KEY AUTO_INCREMENT;" % table_name)
```

Comment:
Dang, there's a new twist - pandas.to_sql seems to convert all ints (even int32) to BIGINT sql type, which can't be used as a primary key.

There is an option on to_sql to pass in a dict mapping columns to SQL types, but that seems like a pain to fill out. It would solve this problem however.